tutorial_3_django_models/slides.md

```diff
@@ -0,0 +1,207 @@
+Tutorial: Create Django Models
+===============================
+[Demonstration time: 9 mins 25 secs (0.785 ~ 79%) | Total time: 12 mins]
+
+Slide 1 [00:08 | 00:08]
+-------------
+Title Slide: ** Creating Django Models **
+
+Slide 2 [00:12 | 00:20]
+--------------
+
+**Learning Objectives**
+
+In this tutorial, we will learn to;
+  - Create a django model
+  - Perform Database migration
+  - Use Admin app
+
+Slide 3 [00:11 | 00:31]
+---------------
+
+**System Requirements**
+  - Ubuntu 16.10
+  - Python 3.5 or higher version
+  - python3.4-venv
+  
+Slide 4 [00:11 | 00:42]
+---------------
+
+**Pre-requisites**
+
+In order to follow this tutorial, you need to know;
+  - how to create a django project
+  - If not, see the relevant django tutorial on http://spoken-tutorial.org
+
+
+Slide 4: What is a Django Model [00:30 | 01:12]
+--------------------------------------
+
+  - Every Django app has one or more models
```

Comment:
A model refers to one particular table for example ```Quiz``` this is different from ```models.py``` which is a collection of models!